core/src/main/java/io/aiven/klaw/controller/TopicController.java

```diff
@@ -36,6 +37,7 @@ public class TopicController {
 
   @Autowired private TopicControllerService topicControllerService;
 
+  @PermissionAllowed(permissionAllowed = {PermissionType.REQUEST_CREATE_TOPICS})
```

Comment:
I was testing removing the REQUEST_CREATE_TOPICS permission, but seems like the code [here](https://github.com/Aiven-Open/klaw/blob/main/core/src/main/java/io/aiven/klaw/validation/TopicRequestValidatorImpl.java#L48) is getting executed. Probably we have to remove the permissions check from there ?

Reply:
Cool thanks I'll take a look at this in the morning, but yeah I will check the ordering of the permission checks and see what it says we can leave it if we want it wont do any harm but at least for testing purposes I can probably nix it